openid-sharedsignals-framework-1_0.md

```diff
@@ -537,42 +548,35 @@ critical_subject_members
 
 > OPTIONAL. An array of member names in a Complex Subject which, if present in
   a Subject Member in an event, MUST be interpreted by a Receiver.
-  
-supported_scopes
 
-> OPTIONAL. A list of OAuth {{RFC6749}} scope names that the Transmitter supports for specific endpoints. The value of this field is a JSON object that has the endpoint names as keys, and arrays of scope name strings they support as their values. OAuth tokens obtained using any of the scopes defined here MUST be accepted by the specified endpoint. Any key that is not defined as an endpoint in the Transmitter Configuration Metadata MUST be ignored. If the supported_scopes member is present in the metadata and an endpoint is not listed as a key, then that endpoint MUST not require OAuth for authorization.
+authorization_schemes
```

Comment:
I'm trying to understand if this is the right way to add protected server metadata. Based on my understanding of the PROTECTEDSERVERMETADATA draft and the example given here: [Protected Resource Metadata Request](https://www.ietf.org/archive/id/draft-jones-oauth-resource-metadata-04.html#name-protected-resource-metadata-) and [Protected Resource Metadata Response](https://www.ietf.org/archive/id/draft-jones-oauth-resource-metadata-04.html#name-protected-resource-metadata-r), I feel we do not need to specify a new field in our metadata. Instead, we should note that implementations can serve a PROTECTEDSERVERMETADATA URL (i.e. /.well-known/oauth-protected-resource) and fields like "authorization_servers" and "scopes_provided" in that metadata to support OAuth.

Reply:
Not sure I am following your point. 

The idea behind adding `authorization_schemes` here was to hint the receivers what to expect. 

> fields like "authorization_servers" and "scopes_provided" in that metadata to support OAuth.

Also, I thought we agreed on decoupling SSF spec from OAuth details like "authorization_servers" and "scopes_provided", as it could be discovered using [Protected Resource Metadata Request](https://www.ietf.org/archive/id/draft-jones-oauth-resource-metadata-04.html#name-protected-resource-metadata-) and [Protected Resource Metadata Response](https://www.ietf.org/archive/id/draft-jones-oauth-resource-metadata-04.html#name-protected-resource-metadata-r)

Reply:
I think my comment was on a [previous commit](https://github.com/openid/sharedsignals/pull/101/commits/b585ac8418d08d9fb80a425b5401f93ea0aa2e4d#diff-193e4a2db2f645f1b4f19c4a9f8facbef29f86e122a8da4c529542750a291a1cR554) that stated this a bit differently. This specific comment can be considered as resolved for now.